numpy/lib/nanfunctions.py

```diff
@@ -548,7 +551,7 @@ def nanprod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     Parameters
     ----------
     a : array_like
-        Array containing numbers whose sum is desired. If `a` is not an
+        Array containing numbers whose product is desired. If `a` is not an
         array, a conversion is attempted.
```

Comment:
Not for the `.accumulate()` method used by the `cumxxx` functions: only one axis.